packages/core/state/selection/selectors.ts

```diff
@@ -76,3 +77,37 @@ export const getUnavailableAnnotationsForHierarchy = createSelector(
             allAnnotations.filter((annotation) => !availableAnnotations.includes(annotation.name))
         )
 );
+
+// COMPOSED SELECTORS
+export const getSortedAnnotations = createSelector(
+    [getAnnotations, getRecentAnnotations],
+    (annotations: Annotation[], recentAnnotationNames: string[]) => {
+        // Create Array of annotations from recentAnnotationNames
+        const recentAnnotations = recentAnnotationNames.flatMap((name) =>
```

Comment:
This could be simpler by having this inner callback return an individual element rather than an array like so:
```
        const recentAnnotations = recentAnnotationNames.flatMap((name) =>
                annotations.find(annotation => annotation.name === name)
        )
        .filter(annotation => !!annotation);
```